db/schema.rb

```diff
@@ -127,8 +127,8 @@
     t.integer  "end_hour",           default: 20
     t.integer  "organization_id"
     t.integer  "ticket_layout",      default: 0
-    t.string   "custom_domain"
     t.integer  "booth_limit",        default: 0
+    t.string   "custom_domain"
```

Comment:
You have quite some unrelated changes. 

You need to use git add -p db/schema.rb

Where you can type 
y - yes for keeping (staging) the change
n - no for not staging the change
s - for splitting the change into smaller chunks (if you want to keep part of it, but not all of the presented chunk)

Please don't add/remove/change anything that is not directly related with your migrations, as it may case problems.

As a second step, you can investigate why / where from those changes originated, to make them disappear and to make sure they will not be added into your commits in the future. I believe the booths related changes in email settings is due to a migration which you changed without reversing or undoing it.

Reply:
Again this is still not fixed.